framework/src/org/checkerframework/common/value/ValueVisitor.java

```diff
@@ -129,6 +134,20 @@ public Void visitAnnotation(AnnotationTree node, Void p) {
     }
 
     @Override
+    protected void commonAssignmentCheck(
+            AnnotatedTypeMirror varType,
+            ExpressionTree valueExp,
+            /*@CompilerMessageKey*/ String errorKey) {
+
+        // This is definitely a hack. However, it does prevent some spurious warnings
+        // from the Value Checker when interacting with unannotated bytecode.
+        if (varType.toString().contains("@")
```

Comment:
varType.toString() won't contain an annotation if the annotation on the varType is invisible.  An annotation is invisible if it has the meta-annotation org.checkerframework.framework.qual.InvisibleQualifier.  `@BottomVal` has this annotation, so that's why this "fixes" that warning.    I would guess this will also cause test failures (or should).